### ISSUE TITLE
add nicer dependency error

### ISSUE THREAD

**Issue by RMHogervorst** [r-quantities/units#157]
I would love it if the failing installation would print which library I have to install. 

In my case libudunits2-dev
I have seen other install error messages that just tell you what you need to install (xml2?)

So in stead of: "if udunits2 is not installed, please install it"

Could we make it something like this:

"Cannot find udunit2, 
for debian / ubuntu use 'sudo apt-get install libudunits2-dev'
for centos7 sudo yum install udunits2-devel
for macos brew install udunits
or equivalent in your distribution. Alternatively, install it from the sources by downloading udunits-2.2.20.tar.gz from ftp://ftp.unidata.ucar.edu/pub/udunits/, and executing the following commands in the download directory:
"
So basically adding the readme bottom part to the error?

**Comment by edzer** [comment]
I entirely agree - I think all packages by @jeroen do this -- we will check this out.

**Comment by RMHogervorst** [comment]
this is what happens in package ssh, I think  it looks neat. Will make a pull request.
```

------------------------- ANTICONF ERROR ---------------------------
Configuration failed because libssh was not found. Try installing:
 * deb: libssh-dev (Debian, Ubuntu, etc)
 * rpm: libssh-devel (Fedora, EPEL)
 * brew: libssh (OSX)
If libssh is already installed, check that 'pkg-config' is in your
PATH and PKG_CONFIG_PATH contains a libssh.pc file. If pkg-config
is unavailable you can set INCLUDE_DIR and LIB_DIR manually via:
R CMD INSTALL --configure-vars='INCLUDE_DIR=... LIB_DIR=...'
--------------------------------------------------------------------
```

**Comment by RMHogervorst** [comment]
oh.. no i will not make a pull request. I have no clue how to change c-code without breaking everything.

**Comment by RMHogervorst** [comment]
Sooo fast!
